wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -1066,15 +1065,15 @@ data PaymentDistribution = PaymentDistribution {
 deriveJSON Serokell.defaultOptions ''PaymentDistribution
 
 instance ToSchema PaymentDistribution where
-  declareNamedSchema =
-    genericSchemaDroppingPrefix "pd" (\(--^) props -> props
-      & ("address" --^ "Address to map coins to.")
-      & ("amount"  --^ "Amount of coin to bind, in Lovelace.")
-    )
+    declareNamedSchema =
+        genericSchemaDroppingPrefix "pd" (\(--^) props -> props
+            & ("address" --^ "Address to map coins to.")
+            & ("amount"  --^ "Amount of coin to bind, in Lovelace.")
+        )
```

Comment:
Indents

Reply:
My bad :no_mouth:
I hadn't yet setup the `.editorconfig` at that time ^^"

Reply:
No worries! Everyone's doing it :sweat_smile: